regression/strings-smoke-tests/java_parselong/test_parselong.java

```diff
@@ -0,0 +1,39 @@
+public class test_parselong
+{
+    public static void main(String[] args)
+    {
+        if (args.length == 1) {
```

Comment:
They do indeed seem to run a bit faster in separate files: ~35 seconds instead of 90 seconds.